Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp

```diff
@@ -1,10 +1,6 @@
-<li ${currentPage == "home" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
-<li ${currentPage == "networks" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/browsenetworks">Networks</a></li>
-<li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/networks">Explore</a></li>
-<li ${currentPage == "networksearch" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/search">Search</a></li>
-<li ${currentPage == "getprojectstatistics" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/statistics">Statistics</a></li>
\ No newline at end of file
+<li ${currentPage == "home" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
+<li ${currentPage == "networks" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/browsenetworks">Networks</a></li>
+<li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/networks">Explore</a></li>
+<li ${currentPage == "exploregraph" ? "class=\"active\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites/${project.unixName}/statistics">Statistics</a></li>
```

Comment:
you want the Statistics menu entry to be active when you are on the statistics page, so you should use your own currentPage value.